.github/workflows/ci-hobbies-helsinki-production.yml

```diff
@@ -2,7 +2,7 @@ name: Hobbies Helsinki Build & Production
 on:
   push:
     tags:
-      - 'release-*'
+      - 'release-hobbies-*'
```

Comment:
production deployments will be done to platta via azure devops pipelines, github prod workflow files will not need

Reply:
ok. I'll add them any way at this point, just for consistency. Those can be removed later

Reply:
or actually, I'll remove them both right a way